src/enterEvent.js

```diff
@@ -0,0 +1,19 @@
+import addTasks from './showTasks';
+import { saveTasks } from './localStorage';
+import { Task, tasks } from './taskClass';
+
+const enterEvent = () => {
+  const inputSelector = document.querySelector('#inputTask');
+
+  inputSelector.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      const task = new Task(inputSelector.value);
+      tasks.push(task);
+      addTasks();
+      saveTasks();
+      inputSelector.value = '';
+    }
+  });
+};
+
+export default enterEvent;
```

Comment:
- [ ] The property `index` have to be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Please fix that